README.md

```diff
@@ -133,6 +141,30 @@ For example, launching the overlay when a button is clicked
 <button onclick="myUI('pub', 'overlay-show');">Search</button>
 ```
 
+### Input box
+
+Search input config allows you to modify the behaviour of the input component.
+
+| Option | Data | Description |
+| :-- | :-: | :-- |
+| `"mode"` | string | Mode of operation for the input element, can be `standard` or `instant` |
```

Comment:
Maybe this could be better?  not clear what standard is?  Is it equivalent to `instantSearch=false` as mentioned in the README online 41?